connaisseur/alert.py

```diff
@@ -53,31 +54,66 @@ class AlertReceiverAuthentication:
     Class to store authentication information for securely sending events to the alert receiver.
     """
 
-    class AlertReceiverBasicAuthentication:
+    authentication_config: dict = None
+    authentication_scheme: str = None
+
+    class AlertReceiverAuthenticationInterface:
+        def __init__(self, alert_receiver_config: dict, authentication_key: str):
+            self.authentication_config = alert_receiver_config.get(authentication_key)
+
+            if self.authentication_config is None:
+                raise ConfigurationError(
+                    f"No authentication configuration found ({authentication_key})."
+                )
+
+            self.authentication_scheme = self.authentication_config.get(
+                "authentication_scheme", self.authentication_scheme
+            )
+            self._validate_authentication_scheme()
+
+        def _validate_authentication_scheme(self) -> None:
+            if not self.authentication_scheme:
+                raise ConfigurationError(
+                    "The authentication scheme cannot be null or empty."
+                )
+
+            if " " in self.authentication_scheme:
+                raise ConfigurationError(
+                    "The authentication scheme cannot contain any space."
+                )
+
+        @abstractmethod
+        def get_header(self) -> dict:
+            pass
+
+    class AlertReceiverNoneAuthentication(AlertReceiverAuthenticationInterface):
+        """
+        Placeholder class for AlertReceiver without authentication.
+        """
+        def __init__(self, alert_receiver_config: dict):
+            pass
+
+        def get_header(self) -> dict:
+            return {}
```

Comment:
This could be the default implementation (instead of `@abstractmethod`). (Just a thought, certainly not mandatory.)